openfisca_us/variables/gov/states/in/tax/income/exemptions/in_additional_exemptions.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_us.model_api import *
+
+
+class in_additional_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN additional exemptions"
+    unit = USD
+    definition_period = YEAR
```

Comment:
Could we add a reference to the Code if available?

Reply:
I added a link to the law in the latest, which is the same for all of the variables. Is there a way to specify subsections in the law separate from the link, or is this enough?

Reply:
If the site doesn't support deeper links (as it looks like it doesn't), please leave a Python comment with the subsection.